config/deb.am

```diff
@@ -38,9 +43,14 @@ if CONFIG_USER
 	pkg7=$${name}-test-$${version}.$${arch}.rpm; \
 	pkg8=$${name}-dracut-$${version}.$${arch}.rpm; \
 	pkg9=$${name}-initramfs-$${version}.$${arch}.rpm; \
+	path_prepend=`mktemp -d /tmp/intercept.XXX`; \
+	ln -s /bin/true $${path_prepend}/dh_shlibdeps; \
```

Comment:
This doesn't seem right, this patch produces .deb packages with no dependencies ... at all:

```
root@debian-9:/usr/src/zfs# dpkg -I libuutil1_0.7.0-0_amd64.deb | egrep '(Package|Depends):'
 Package: libuutil1
```

compared to the "original" (without this patch) resulting package:

```
root@debian-9:/usr/src/zfs# dpkg -I libuutil1_0.7.0-0_amd64.deb | egrep '(Package|Depends):'
 Package: libuutil1
 Depends: libc6 (>= 2.14), libuuid1 (>= 2.16), zlib1g (>= 1:1.1.4)
```